posthog/hogql_queries/web_analytics/session_attribution_explorer_query_runner.py

```diff
@@ -11,7 +11,25 @@
     HogQLFilters,
 )
 
-BREAKDOWN_NULL_DISPLAY = "(none)"
```

Comment:
Is this just extra? It also exists on the `stats_table.py`.

Reply:
Yeah, we copied over by mistake